### PR TITLE
Pay 66029 fi merchant limit 2000

### PR DESCRIPTION
**Pull request by TadasAmbrazaitis**

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Subscriptions API Release Notes
+<div class='post-date'>26 August 2020 - amount</div> 
+Added limit to payment and oneoffpayment amount for FI merchants to 2000.
```
Comment (ReisEdgar) [inline docs/release_notes.md:3]: Maybe "oneoffpayment" could be written as "one off payment" ?

Comment (granatoidas) [reply docs/release_notes.md:3]: better yet "one-off payment"

Comment (granatoidas) [reply docs/release_notes.md:3]: also have we really "added" limit or "updated" it?

Comment (TadasAmbrazaitis) [reply docs/release_notes.md:3]: the limit was already there, so we actually didn't do anything

**Review by granatoidas — APPROVED** [review]

**Pull request merged**